tests/integration/targets/ntnx_protection_policies_v2/tasks/protection_policies.yml

```diff
@@ -0,0 +1,1522 @@
+---
```

Comment:
Please add here prequistes here like what vars you are expecting and which task so that it can be prepped before running